### PR TITLE
chore(scripts/RuinsOfAhnQiraj): cleanup ruins_of_ahnqiraj.h

### PR DESCRIPTION
**Pull request by Si1ker**
## Changes Proposed:
- Remove unused code
- Remove duplicates

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Maelthyrr** [comment]
Need to be update since the update to the new script system.

**Comment by Si1ker** [comment]
> Need to be update since the update to the new script system.

@Maelthyrr Done.

**Comment by Si1ker** [comment]
@Maelthyrr labels?
@IntelligentQuantum approve?

File: src/server/scripts/Kalimdor/RuinsOfAhnQiraj/boss_ayamiss.cpp [hunk 1]
```diff
@@ -180,8 +180,7 @@ struct boss_ayamiss : public BossAI
                 {
                     DoCast(target, SPELL_PARALYZE, true);
                     instance->SetGuidData(DATA_PARALYZED, target->GetGUID());
-                    uint8 index = urand(0, 1);
-                    me->SummonCreature(NPC_HIVEZARA_LARVA, LarvaPos[index], TEMPSUMMON_TIMED_OR_CORPSE_DESPAWN, 30000);
+                    DoCastAOE(RAND(SPELL_SUMMON_LARVA_A, SPELL_SUMMON_LARVA_B), true);
```
Comment (Nyeriah) [inline src/server/scripts/Kalimdor/RuinsOfAhnQiraj/boss_ayamiss.cpp:183]: Something seems to have happened here, merge conflict probably went wrong, not sure if it is just me

image

File: src/server/scripts/Kalimdor/RuinsOfAhnQiraj/boss_ayamiss.cpp [hunk 2]
```diff
@@ -201,8 +200,13 @@ struct boss_ayamiss : public BossAI
...
             {
-                Position Pos = me->GetRandomPoint(SwarmerPos, 80.0f);
-                me->SummonCreature(NPC_HIVEZARA_SWARMER, Pos, TEMPSUMMON_CORPSE_TIMED_DESPAWN, 5000);
+                Position const offset = { 0.0f, 0.0f, 20.0f, 0.0f };
+                Position spawnpos = me->GetRandomPoint(SwarmerPos, 80.0f);
+                spawnpos.RelocateOffset(offset);
+                if (Creature* wasp = me->SummonCreature(NPC_HIVEZARA_SWARMER, spawnpos, TEMPSUMMON_CORPSE_TIMED_DESPAWN, 5000))
+                {
+                    wasp->GetMotionMaster()->MoveRandom(10.0f);
+                }
```
Comment (Nyeriah) [inline src/server/scripts/Kalimdor/RuinsOfAhnQiraj/boss_ayamiss.cpp:209]: here too, event will never repeat, merge conflict def went wrong

**Comment by Si1ker** [comment]
@Nyeriah ?????

any complaints about what's already in the core go for your pr on ayamiss.
I just removed unused stuff and renamed it to (old, first AC commit ever) already existing values.

**Comment by Nyeriah** [comment]
I posted images of what I saw, and the merge had issues

I see that those issues were either corrected already or GitHub had issues displaying the merge PR

I appreciate your willingness to help, but unfurtunately I see there's more effort spent on snarky remarks than addressing the concerns and questions raised by whoever reviews your PRs

**Review by Nyeriah — APPROVED** [review]

**Pull request merged**